helm/kafka/scripts/aws-issue-certificate.sh

```diff
@@ -0,0 +1,48 @@
+set -e
+
+dir=/timecard-restapi-keystore
+alias=$1
+days=100
```

Comment:
Ln 19 in the script - Checks if it is going to expire that day - "if openssl x509 -checkend 86400 -noout -in $service_alias-certificate.pem" 

If this is the case it will generate new certs. But happy to increase the number of days if we deem this necessary.

Reply:
Sorry I misread the script.
So line 19 checks if the certificate exists and is still valid, and successfully exits if that is the case. Otherwise, it will delete all the previously generated certificate/key and keystore and regenerates them again with refreshed 100 days validity. Am I right in my interpretation?
If so, and given that the script is run only when there is a deployment, if there hasn't been a deployment for more than a 100 days (which is plausible once a service is mature enough), the certificate would expire.
I am not suggesting we should increase the certificate validity, but maybe there are Home Office guidelines around certificate validity and rotation we should be adhering to?
Also we should look into a mechanism to rotate these credentials (e.g. daily cronjob/workflow which checks certificate and regenerates if about to expire).
I think for now this can be resolved. but we should track this in a separate task to ensure we already have a certificate rotation mechanism in place by the time it's due to expire 
@stephenkaye-ho FYI

Reply:
I get the point. We would need to have a seperate task to ensure we rotate the and update the certs.

Reply:
SSL certificate rotation to be addressed separately in this task: https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-2606